.vscode/launch.json.jinja

```diff
@@ -1,34 +0,0 @@
-{%- import "_macros.jinja" as macros -%}
-{
```

Comment:
After deleting this file, copier gets the `launch.json` file used by the template itself and copies it into the subproject.

That's an error. You'll have to add that file to exclusions in `copier.yml`.